### ISSUE TITLE
"Vent Front" climate entity no longer provided

### ISSUE THREAD

**Issue by madd0** [vincentwolsink/home_assistant_micronova_agua_iot#171]
Hi

I recently had a Nobis A11 Glass Round Top pellet stove installed. I was very pleased to find this integration which recognized it immediately! Thanks for your work!

However, the integration suddenly stopped providing the entity that allows fan control :/

I've taken a quick look at the code, but haven't had time to really understand it. I'm guessing it probably has something to do with the data provided by the API for this specific stove. Here are the diagnostic data:

[config_entry-aguaiot-01JCZRGQN524Q106WJGYJ09BE6.json](https://github.com/user-attachments/files/17844476/config_entry-aguaiot-01JCZRGQN524Q106WJGYJ09BE6.json)

Is there anything else I can provide or do to try to resolve this issue?

Thanks!

**Comment by madd0** [comment]
Yes, it was. It worked for a day or two, and then became unavailable.

**Comment by vincentwolsink** [comment]
Please try [0.6.0](https://github.com/vincentwolsink/home_assistant_micronova_agua_iot/releases/tag/0.6.0)
